README.md

```diff
@@ -16,12 +16,106 @@ make env
 
 ## Development Server
 To run the development server, create a `local_settings.py` file as a sibling of `settings.py`. Add a `SECRET_KEY` value
-with some random stuff (e.g `SECRET_KEY = 'sjfio3u903RaggleFraggle'`).
+with some random stuff (e.g `SECRET_KEY = 'sjfio3u903RaggleFraggle'` and see "Environment Variables" section).
 
 The Django local development can be run via:
 
 ```bash
 make watch
 ```
 
+Another means to run local is the manage.py from within the wellregistry path:
 
+```bash
+python -m manage runserver
+```
+Note that environment variable must be configured to connect to a postgres database.
+
+To run tests locally:
+
+```bash
+python -m manage test
+```
+
+
+## Environment Variables
+
+    This project has a private companion config project where environment variable
+    files for each deployment tier are maintained. The values in the config files are
+    applied to the docker container for each teir at deployment. 
+    The following sub sections will describe the various variables used.
+
+General
+
+    DEBUG: boolean - true for debug level logging
+
+Cloud
+
+    These environment variables are required for the cloud deployment.
+    They are not used for local development.
+
+        SECRET_KEY: AWS secret key
+        ALLOWED_HOSTS: mapped URL
+        CIDR_RANGES: IP ranges allowed to connect
+
+Database (root)
+
+    The DATABASE values are defined to connect to cloud RDS or local DB installation.
+    These values are used by the initial Django migrations *only* to configure the application.
+
+        DATABASE_NAME: posgres database name
+        DATABASE_HOST: posgres host url 
+            - use 'localhost' for local development database
+            - use the actual tier database host URL for cloud deployment
+        DATABASE_PORT: optional posgres port - if not specified then default is '5432' 
+        DATABASE_USERNAME: postgres root user name
+        DATABASE_PASSWORD: postgres root password
+
+Database (app)
+
+    The following should be set to initial values for a new database configuration.
+    Once the database initial migration scripts are run then they need to be set to 
+    expected values. While these can be arbitrarily configurable for local database 
```

Comment:
This paragraph is confusing. Do these values change after you initialize the database? Seems like they should not be the same environment variables then?

Reply:
Right, I was not sure how to articulate this. I will give it some thought and word smith.

Reply:
I updated the documentation. I revised the direction and added more sentences. It might be confusing because I am describing something that you and I already know. I am just trying to get across the dual nature of the environment variables: used to both setup and utilize the database.